src/main/java/com/datapipe/jenkins/vault/VaultAccessor.java

```diff
@@ -19,23 +22,30 @@
     private transient VaultConfig config;
 
 
+
     public void init(String url) {
-        init(url, null, false);
+        init(url, null, DEFAULT_TRUSTSTORE, false);
     }
 
     public void init(String url, boolean skipSslVerification) {
-        init(url, null, skipSslVerification);
+        init(url, null, DEFAULT_TRUSTSTORE, skipSslVerification);
     }
 
     public void init(String url, VaultCredential credential) {
-        init(url, credential, false);
+        init(url, credential, DEFAULT_TRUSTSTORE, false);
     }
 
     public void init(String url, VaultCredential credential, boolean skipSslVerification) {
+        init(url, credential, DEFAULT_TRUSTSTORE, skipSslVerification);
+    }
+
+    public void init(String url, VaultCredential credential, File trustStore, boolean skipSslVerification) {
```

Comment:
Oh no, I cannot handle this :)

We need a PR that changes this to using VaultConfig directly or at least a builder pattern.

Reply:
Done.